### PR TITLE
Correcting inconsistencies and formatting in query names and folders Closes #1893

### LINKED ISSUE #1893: Correcting inconsistencies and formatting in query names and folders
### Description
This issue relates to the correcting of inconsistencies between query names in different documentation, folder names and formatting.

### PR DESCRIPTION
**Pull request by rafaela-soares**
Closes #1893

### CODE REVIEW

File: assets/queries/terraform/aws/cloudwatch_metrics_not_enabled/metadata.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "081069cb-588b-4ce1-884c-2a1ce3029fe5",
+  "queryName": "CloudWatch Metrics Not enabled",
```
Comment (jorge-cruz) [inline assets/queries/terraform/aws/cloudwatch_metrics_not_enabled/metadata.json:3]:
```suggestion
  "queryName": "CloudWatch Metrics Not Enabled",
```

File: assets/queries/terraform/aws/eks_cluster_has_public_access/metadata.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "42f4b905-3736-4213-bfe9-c0660518cda8",
+  "queryName": "EKS cluster Has Public Access",
```
Comment (jorge-cruz) [inline assets/queries/terraform/aws/eks_cluster_has_public_access/metadata.json:3]:
```suggestion
  "queryName": "EKS Cluster Has Public Access",
```

File: assets/queries/terraform/aws/eks_cluster_has_public_access_cidrs/metadata.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "61cf9883-1752-4768-b18c-0d57f2737709",
+  "queryName": "EKS cluster Has Public Access CIDRs",
```
Comment (jorge-cruz) [inline assets/queries/terraform/aws/eks_cluster_has_public_access_cidrs/metadata.json:3]:
```suggestion
  "queryName": "EKS Cluster Has Public Access CIDRs",
```

File: assets/queries/terraform/aws/google_kms_crypto_key_rotation_period_not_recommended/metadata.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "d8c57c4e-bf6f-4e32-a2bf-8643532de77b",
+  "queryName": "Google KMS Crypto Key Rotation Period Not Recommended",
+  "severity": "MEDIUM",
+  "category": "Encryption & Key Management",
+  "descriptionText": "Make sure Encryption keys changes after 90 days",
```
Comment (jorge-cruz) [inline assets/queries/terraform/aws/google_kms_crypto_key_rotation_period_not_recommended/metadata.json:6]:
```suggestion
  "descriptionText": "Make sure Encryption keys change after 90 days",
```

File: assets/queries/terraform/aws/iam_policies_allow_all_in_policy_statement/metadata.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "575a2155-6af1-4026-b1af-d5bc8fe2a904",
+  "queryName": "IAM Policies Allows All ('*') In Policy Statement",
```
Comment (jorge-cruz) [inline assets/queries/terraform/aws/iam_policies_allow_all_in_policy_statement/metadata.json:3]:
```suggestion
  "queryName": "IAM Policy Allows All ('*') In Policy Statement",
```

File: assets/queries/terraform/aws/neptune_cluster_not_encrypted/metadata.json [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
-	"severity": "MEDIUM",
-	"category": "Encryption & Key Management",
-	"descriptionText": "Check if Neptune Cluster Storage is securely encypted",
-	"descriptionUrl": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/neptune_cluster#storage_encrypted",
-	"platform": "Terraform"
+  "id": "98d59056-f745-4ef5-8613-32bca8d40b7e",
+  "queryName": "Neptune Cluster Not Encrypted",
+  "severity": "MEDIUM",
+  "category": "Encryption & Key Management",
+  "descriptionText": "Check if Neptune Cluster Storage is securely encypted",
```
Comment (jorge-cruz) [inline assets/queries/terraform/aws/neptune_cluster_not_encrypted/metadata.json:6]:
```suggestion
  "descriptionText": "Check if Neptune Cluster Storage is securely encrypted",
```

File: assets/queries/terraform/aws/sqs_with_sse_disabled/metadata.json [hunk 7]
```diff
@@ -1,9 +1,9 @@
 {
-	"id": "6e8849c1-3aa7-40e3-9063-b85ee300f29f",
-	"queryName": "SQS with SSE disabled",
-	"severity": "HIGH",
-	"category": "Network Security",
-	"descriptionText": "Amazon Simple Queue Service (SQS) queue is not protecting the contents of their messages using Server-Side Encryption (SSE)",
-	"descriptionUrl": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue",
-	"platform": "Terraform"
+  "id": "6e8849c1-3aa7-40e3-9063-b85ee300f29f",
+  "queryName": "SQS With SSE disabled",
```
Comment (jorge-cruz) [inline assets/queries/terraform/aws/sqs_with_sse_disabled/metadata.json:3]:
```suggestion
  "queryName": "SQS With SSE Disabled",
```

File: assets/queries/terraform/azure/postgre_sql_server_without_connection_throttling/metadata.json [hunk 8]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "2b3c671f-1b76-4741-8789-ed1fe0785dc4",
+  "queryName": "PostgreSQL Server Without Connection Throttling",
+  "severity": "MEDIUM",
+  "category": "Logging",
+  "descriptionText": "Make sure that for PostgreSQL server, Connection Throttling is set",
```
Comment (jorge-cruz) [inline assets/queries/terraform/azure/postgre_sql_server_without_connection_throttling/metadata.json:6]:
```suggestion
  "descriptionText": "Ensure that Connection Throttling is set for the PostgreSQL server",
```

**Review by jorge-cruz — CHANGES_REQUESTED** [review]

**Review by jorge-cruz — DISMISSED** [review]

**Comment by rogeriopeixotocx** [comment]
I used this script to rename all query dirs to lowercase and life is good:

```
#!/usr/bin/env python3
import glob
import os

def joinp(*arr):
  return os.sep.join(arr)

glob_result = []
glob_result = glob.glob(joinp("assets","queries","ansible","**", "*"))
glob_result.extend(glob.glob(joinp("assets","queries","terraform","**", "*")))
glob_result.extend(glob.glob(joinp("assets","queries","cloudFormation","**")))
glob_result.extend(glob.glob(joinp("assets","queries","dockerfile","**")))
glob_result.extend(glob.glob(joinp("assets","queries","k8s","**")))
dirs = [path for path in glob_result if os.path.isdir(path)]
for directory in dirs:
  base_dir = os.sep.join(directory.split(os.sep)[:3])
  query_dir = os.sep.join(directory.split(os.sep)[3:])
  if any(x.isupper() for x in query_dir):
    lower_query_dir = joinp(base_dir, query_dir.lower())
    print(f"renaming {directory} to {lower_query_dir}")
    os.rename(directory, lower_query_dir)
```
cc: @ruigomescx @rafaela-soares

**Review by jorge-cruz — APPROVED** [review]

**Pull request merged**